{"name": "Проверка, что с несуществующим пользователем, запрос возвращает ошибку", "status": "passed", "start": 1724995082008, "stop": 1724995082898, "uuid": "0361a9a9-c160-4cc9-9a6a-94768145b3db", "historyId": "aceb1830b6f1fdc26311d0ae57f94742", "testCaseId": "aceb1830b6f1fdc26311d0ae57f94742", "fullName": "tests.test_login.TestLogin#test_login_with_nonexistent_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-DPIGL2N"}, {"name": "thread", "value": "12736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}